src/components/dialogs/ConfirmTxDialog/ConfirmTxDialog.scss

```diff
@@ -51,3 +51,22 @@
 :global(.TokenValue__root) {
```

Comment:
=(
This style will be the global for the whole app
All :global selectors should be nested to the scoped parents